packages/chain-adapters/src/bitcoin/BitcoinChainAdapter.ts

```diff
@@ -458,7 +458,7 @@ export class ChainAdapter implements IChainAdapter<ChainTypes.Bitcoin> {
           txid: msg.txid
         })
       },
-      (err) => onError({ message: err.message })
+      (err: chainAdapters.SubscribeError) => onError({ message: err.message })
```

Comment:
Should not need this type and it is also the incorrect type being returned by `subscribeTxs` in unchained